truffle/com.oracle.truffle.examples/src/com/oracle/truffle/example/simplegraphanalitycs/TruffleExt.java

```diff
@@ -0,0 +1,77 @@
+package com.oracle.truffle.example.simplegraphanalitycs;
+
+import com.oracle.truffle.api.CallTarget;
+import com.oracle.truffle.api.RootCallTarget;
+import com.oracle.truffle.api.Truffle;
+import com.oracle.truffle.api.TruffleLanguage;
+import com.oracle.truffle.api.frame.MaterializedFrame;
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.instrument.Visualizer;
+import com.oracle.truffle.api.instrument.WrapperNode;
+import com.oracle.truffle.api.nodes.Node;
+import com.oracle.truffle.api.nodes.RootNode;
+import com.oracle.truffle.api.source.Source;
+import java.io.IOException;
+import java.util.function.Supplier;
+
+public class TruffleExt {
+    
+    public static RootCallTarget createCallTarget(Supplier r) {
+        RootCallTarget rootTarget = Truffle.getRuntime().createCallTarget(new RootNode(TruffleLanguage.class, null, null) {
+            @Override
+            public Object execute(VirtualFrame vf) {
+                return r.get();
+            }
+        });
+        return rootTarget;
+    }
+    
+    public class TestLanguage extends TruffleLanguage {
+
+        @Override
+        protected Object createContext(TruffleLanguage.Env env) {
+            throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
I don't like the comment, but actually I think this class is unnecessary (is it even used). If you need a dummy `TruffleLanguage` class, I think you can use the existing `com.oracle.truffle.api.TestingLanguage`.